java/arcs/android/common/resurrection/ResurrectionRequest.kt

```diff
@@ -30,6 +30,12 @@ data class ResurrectionRequest(
     val componentType: ComponentType,
     val intentAction: String?,
     val intentExtras: PersistableBundle?,
+    /**
+     * A string, which along with [componentName], uniquely identifies this request. When
+     * clients are resurrected, the client receives a set of [StorageKeys] supplied in
+     * [notifyOn] along with the [notifierId] used to create the request.
+     */
+    val notifierId: String,
```

Comment:
Yeah, I'm definitely thinking `notifierId` is a misnomer here, and "targetId" or something would be better - since the arc host isn't what's doing the notifying